test/client-node-max-header-size.js

```diff
@@ -0,0 +1,22 @@
+'use strict'
+
+const { execSync } = require('node:child_process')
+const { test } = require('tap')
+
+const command = 'node -e "require(\'./index-fetch\').fetch(\'https://httpbin.org/get\')"'
```

Comment:
If you want to test it under fetch, you need to put the testing suite under `test/fetch` as that suite runs with `Node v16` and above.

I missed that at a first glance, but it would be good to have a new one just for `fetch` and a second one that does the same but with pure `undici`.

Reply:
OK, copied the test over to `test/fetch`, what do you mean by a pure `undici` test?

Reply:
test that this behaves the same with the `request` method or something like that.